src/main/resources/migrations/00043_add_payouts_foreign_key_to_transaction.sql

```diff
@@ -0,0 +1,5 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_payouts_foreign_key_to_transaction
+ALTER TABLE transaction ADD COLUMN gateway_payout_id VARCHAR(26) REFERENCES payouts(gateway_payout_id);
```

Comment:
I think we need to increase size of the column as Stripe payout IDs are more than 26 characters ex: `po_1GPf0PJqIiWTNsILGMw50leq`. May be 50 chars or so ? so we don't need to keep changing if Stripe varies length